roles/openshift_logging/vars/main.yaml

```diff
@@ -3,10 +3,8 @@ openshift_master_config_dir: "{{ openshift.common.config_base }}/master"
 es_node_quorum: "{{openshift_logging_es_cluster_size|int/2 + 1}}"
 es_min_masters_default: "{{ (openshift_logging_es_cluster_size | int / 2 | round(0,'floor') + 1) | int }}"
```

Comment:
This calculation should end up being the same as `es_node_quorum`.  So we should probably merge this with `node_quorum` and use that instead eliminating this parameter.